mautic/mautic
# [UX] Change “Render style from Template?” to “Use theme style?” when creating form

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
The current label “Render style from Template?” presents an opportunity for enhancement. It would be beneficial to make the label more precise and aligned with the rest of the platform’s terminology. The proposed solution in this pull request is to change the label to “Use theme style?”, which directly relates to the improvement desired.

The change primarily benefits users who interact with the form creation process, providing them with a clearer understanding of the options available. By implementing this change, we can expect a more intuitive user experience and a more consistent user interface across the platform. This could potentially lead to users being more comfortable with using the platform and exploring its features.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to Forms
3. Create a new form
4. Look at the new title on the right side 

![image](https://github.com/mautic/mautic/assets/116097999/c0ae7c6d-39e9-4c79-b38e-60fb40ac0c36)
